style.css

```diff
@@ -154,6 +154,19 @@ html, body {
     background: white;
     min-height: 100vh;
     padding: 50px;
+.external_website {
+	max-width: var(--page-width);
+	width: 100vh;
```

Comment:
Maybe it's best to remove this line (`width: 100vh;`), otherwise it seems to sit outside the margins.
![image](https://github.com/user-attachments/assets/15656da5-f3ec-491d-84cc-73867d7dd32f)